analysis/staticAnalysisReport.Rmd

```diff
@@ -0,0 +1,171 @@
+---
```

Comment:
Please name the file `static-analysis-report.Rmd`

Reply:
Alright!